lms/migrations/versions/7e4124035651_add_the_provisioning_column.py

```diff
@@ -0,0 +1,31 @@
+"""
+Add the provisioning column.
```

Comment:
Comment string might be more helpful if it explained what the column is for—_we_ know, but future devs might not immediately understand that this is a feature-enabling toggle.

Reply:
Are future devs going to be looking at the code of past DB migrations to find out what model attributes are for though, I wonder? :)

Reply:
I have totally done exactly that before :)